src/CarbonAware.WebApi/src/Models/CarbonIntensityBaseDTO.cs

```diff
@@ -0,0 +1,23 @@
+namespace CarbonAware.WebApi.Models;
+
+using System.Text.Json.Serialization;
+
+[Serializable]
+public record CarbonIntensityBaseDTO
+{
+    /// <summary>the azure location name </summary>
+    /// <example> eastus </example>
+    [JsonPropertyName("location")]
+    public string Location { get; set; } = string.Empty;
+
+    /// <summary>the time at which the job actually started </summary>
+    /// <example>2022-03-01T15:30:00Z</example>
+    [JsonPropertyName("jobStartTime")]
+    public DateTimeOffset JobStartTime { get; set; }
```

Comment:
I know we keep talking about param namings but "job" seems specific to "Azure Batch Jobs" versus getting the actual carbon intensity for a time period. I.e., most people don't think in terms of "jobs", that seems more specific to this case.

Reply:
Any suggestion for a name then?? StartTime and EndTime seem too generic and confusing based on feedback from other specs and jobStartTime gave me a better idea of what the startTime is for. Maybe activityStartTime or workflowStartTime would be better?

Reply:
I think StartTime and EndTime work for this spec.  It was confusing in the other spec due to fact that there were multiple start and end times (ie start/end of batch; start/end of forecast).  With this spec we know that the only start and end time we are discussing is the compute times.